basic/src/test/java/org/jboss/hal/testsuite/test/configuration/elytron/authentication/ElytronAuthenticationOperations.java

```diff
@@ -0,0 +1,50 @@
+package org.jboss.hal.testsuite.test.configuration.elytron.authentication;
+
+import org.wildfly.extras.creaper.core.online.ModelNodeResult;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ElytronAuthenticationOperations {
+
+    private OnlineManagementClient client;
+
+    public ElytronAuthenticationOperations(OnlineManagementClient client) {
+        this.client = client;
+    }
+
+    /**
+     * Back ups values of attributes, performs and {@link Action} and reverts the attributes to their original value.
+     *
+     * @param address        Address of attributes
+     * @param action         Action performed with backed up attributes
+     * @param attributes     Names of attributes
+     */
+    public void performActionOnAttributeAndRevertItsValueToOriginal(Address address, Action action, String... attributes) throws Exception {
```

Comment:
Do you really plan to use attributes as String array? Since in the PR you always pass just one String. I'm asking since if you use `String attribute` parameter instead of `String... attributes` array parameter you can switch the order of the last 2 parameters (in that case parameters would be `Address address, String attribute, Action action`) and the usage of this method would be IMHO much more readable without attribute name lost somewhere after lambda body ... just suggestion.